### ISSUE TITLE
[BUG] Upgrade log packaging format error

### ISSUE THREAD

**Issue by starbops** [harvester/harvester#4500]
**Describe the bug**

When upgrading from v1.1.2 to v1.2.0-rc6, downloaded upgrade log files don't get handled correctly. For example:

```
...
2023-09-07T07:47:09+00:00	cattle-system.rancher-64f6b99dfb-w6h6c.rancher	{"stream":"stdout","logtag":"F","message":"2023/09/07 07:47:09 [INFO] Rancher version v2.7.5 (eef55334b) is starting","kubernetes":{"pod_name":"rancher-64f6b99dfb-w6h6c","namespace_name":"cattle-system","pod_id":"06762e27-89a8-4f54-9499-b74c11a69c5f","labels":{"app":"rancher","pod-template-hash":"64f6b99dfb","release":"rancher"},"annotations":{"cni-projectcalico-org/containerID":"eb8003158ff88c75ed625639ae05d2b13287719b3e3f6d8d50e1c87ea4d6032b","cni-projectcalico-org/podIP":"10.52.0.97/32","cni-projectcalico-org/podIPs":"10.52.0.97/32","harvesterhci-io/timestamp":"2023-09-07T07:00:37Z","k8s-v1-cni-cncf-io/network-status":"[{\n    \"name\": \"k8s-pod-network\",\n    \"ips\": [\n        \"10.52.0.97\"\n    ],\n    \"default\": true,\n    \"dns\": {}\n}]","k8s-v1-cni-cncf-io/networks-status":"[{\n    \"name\": \"k8s-pod-network\",\n    \"ips\": [\n        \"10.52.0.97\"\n    ],\n    \"default\": true,\n    \"dns\": {}\n}]","kubernetes-io/psp":"global-unrestricted-psp"},"host":"harvester-9v9tt","container_name":"rancher","docker_id":"4277f0873fb6a53218b927f657dfe93e0545e62fe89004409b69ca188f879c84","container_hash":"sha256:81ee0878ffcdc37e87f0c4a4b4d528763ad24442aaf07475370d15e1d887cb54","container_image":"docker.io/rancher/rancher:v2.7.5"}}
...
```

The log line is in JSON format with many unneeded metadata. However, the only log message we want is the following:

```
...
2023/09/07 07:47:09 [INFO] Rancher version v2.7.5 (eef55334b) is starting
...
```

Note that though this mis-behavior does not impact the upgrade progress, it may increase the likelihood of the upgrade log downloading problem mentioned in #4222.

**To Reproduce**
1. Prepare a Harvester cluster in v1.1.2 (single node should be enough)
2. Upgrade to v1.2.0-rc6
3. Click the "Download Log" button on the upgrade dialog after **Upgrading System Service** phase is passed.
4. Extract the downloaded archive, you should be able to see the log content in JSON format

**Expected behavior**

The downloaded log be in the correct format.

**Support bundle**

[supportbundle_dcd840f1-aed4-45cc-9151-9688fde766c1_2023-09-07T09-23-54Z.zip](https://github.com/harvester/harvester/files/12547151/supportbundle_dcd840f1-aed4-45cc-9151-9688fde766c1_2023-09-07T09-23-54Z.zip)

**Environment**
 - Harvester ISO version: v1.1.2 to v1.2.0-rc6
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): KVM

**Additional context**

Clicking "Download Log" before the upgrade progress entering the **Upgrading System Service** phase is not affected.

**Comment by starbops** [comment]
The bug only occurs in the current transitions, i.e., upgrading from v1.1.2 to v1.2.0, or v1.1.2 to v1.1.3. It won't happen if we pass these points. With that said, it may be better to put the packaging script into the `harvester-upgrade` image, just like the `node_upgrade.sh` script.

**Comment by TachunLin** [comment]
Verifing **fixed** on `v1.2-7007cb37-head` (09/08). Close this issue.

### Test Result

- [x] Upgrading from v1.1.2 to v1.2-head

   - [x] Before upgrading system service 
     - [x]  The the downloaded log content should be in `*.log` format and be wrapped in a total *.zip file
       <details><summary>Details</summary>
       
          ![image](https://github.com/harvester/harvester/assets/29251855/648b4c3d-5c7d-4ada-b85d-17252d7c041b)
          
          ![image](https://github.com/harvester/harvester/assets/29251855/9342d9fe-862c-4a23-bb8c-8676ac2bc26a)
       </details>
     - [x]  The the downloaded log content, should **not be** in the JSON format
     
     
       <details><summary>Details</summary>
       
          ![image](https://github.com/harvester/harvester/assets/29251855/100dd637-8830-4153-ae6a-28503952622b)

       </details>


   - [x] After upgrading system service 
     - [x]  The the downloaded log content should be in `*.log` format and be wrapped in a total *.zip file
       <details><summary>Details</summary>
       
         ![image](https://github.com/harvester/harvester/assets/29251855/f2e79f50-8b08-4e7a-a834-6714f498710d)
         
        ![image](https://github.com/harvester/harvester/assets/29251855/5bbb26b1-115d-431a-8bae-29577ffb9238)
       </details>
     

     - [x]  The the downloaded log content, should not be in the JSON format

       <details><summary>Details</summary>
       
          ![image](https://github.com/harvester/harvester/assets/29251855/068253ea-d223-435c-be2a-ae9574920977)

       </details>


- [x] Same version upgrade (V1.2-head)

   - [x] Before upgrading system service 
     - [x]  The the downloaded log content should be in `*.gzip` format and be wrapped in a total *.zip file

       <details><summary>Details</summary>
       
        ![image](https://github.com/harvester/harvester/assets/29251855/b3b14c7b-cfdf-4f7f-8f00-c3d7f1f97935)
        ![image](https://github.com/harvester/harvester/assets/29251855/2cc32614-d050-4a69-a239-742cdf196ab2)
        ![image](https://github.com/harvester/harvester/assets/29251855/0207d3f3-aaab-48b7-847a-e8d6fd63ed5d)

       </details>




     - [x]  The the downloaded log content, should not be in the JSON format


       <details><summary>Details</summary>
       
        ![image](https://github.com/harvester/harvester/assets/29251855/47ef504a-e833-416f-bcd5-7ead55f2a818)

       </details>



   - [x] After upgrading system service 
     - [x]  The the downloaded log content should be in `*.gzip` format and be wrapped in a total *.zip file

       <details><summary>Details</summary>
       
        ![image](https://github.com/harvester/harvester/assets/29251855/9ce7acba-c8df-4f74-82e6-57ad8b24b4b5)
        
        ![image](https://github.com/harvester/harvester/assets/29251855/2cb9f56e-cc76-493a-8354-e1708922f9c7)
        
        ![image](https://github.com/harvester/harvester/assets/29251855/f577e365-5063-4224-adc0-5826c5cbe7d9)

       </details>




     - [x]  The the downloaded log content, should not be in the JSON format

       <details><summary>Details</summary>
       
        ![image](https://github.com/harvester/harvester/assets/29251855/90f21e3b-651d-4ab0-9497-d8ce19f3460b)

       </details>


### Test Information
* Test Environment: single nodes harvester on local kvm machine
* Harvester version: `v1.2-7007cb37-head` (09/08)

### Verify Steps
<details><summary>Execution Steps</summary>

  #### Upgrading from v1.1.2 to v1.2-head 
  
  1. Prepare a v1.1.2 Harvester cluster (single node would be fine)
  2. Upgrade with the ISO containing the fix
  3. Click the "Download Log" button on the upgrade dialog before entering Upgrading System Service phase
  4. Check the downloaded file are in *.log format and wrapped in a *.zip file
  5. Check the downloaded log content, should not be in the JSON format
  6. Wait until the upgrade passed the Upgrading System Service phase
  7. Click the "Download Log" button again
  8. Check the downloaded log content, should not be in the JSON format
  9. Check the downloaded file are in *.log format and wrapped in a *.zip file

#### Same version upgrade (V1.2-head)

  1. Prepare a Harvester cluster provisioned with the ISO containing the fix
  2. Upgrade with the same ISO
  3. Click the "Download Log" button **before** and **after** the Upgrading System Service phase 
  4. Check the downloaded log content, should be in gzip format
  5. Check the downloaded file are in *.gz format and wrapped in a *.zip file


</details>
